### PR TITLE
Add client install guides

### PR DESCRIPTION
**Pull request by rmeissner**
- Added install guides for clients
- Minor service updates

### CODE REVIEW

File: source/services/history.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # History Service
```
Comment (Uxio0) [inline source/services/history.md]: Should it be Transaction Service?

File: source/clients/chrome_extension.md [hunk 2]
```diff
@@ -1,3 +1,34 @@
...
+
+## Download Rinkeby Version
+
+The Rinkeby version of the Gnosis Safe is only available from the GitHub releases page. This version will not receive automatic updates.
+
+[View on GitHub releases](https://github.com/gnosis/safe-browser-extension/releases) (zip file in the assets prefixed with `-rinkeby`)
+
+## Install Extension from GitHub
+
+1. Download the zip file for the corresponsing version from the [GitHub releases](https://github.com/gnosis/safe-browser-extension/releases)
```
Comment (germartinez) [inline source/clients/chrome_extension.md]: corresponding*

File: source/clients/android.md [hunk 3]
```diff
@@ -1,3 +1,26 @@
 # Android Client (Safe Personal Edition)
 
+## Install Mainnet Version
+
+The Mainnet version of the Gnosis Safe Android app can be downloaded from the Google PlayStore or from the GitHub releases page.
+
+Note: The versions from GitHub and the PlayStore are differently signed, therefore it is not possible to update the PlayStore version with a version downloaded from GitHub and vise-versa. Also the GitHub version will not recive automatic updates.
+
+[View on PlayStore](https://play.google.com/store/apps/details?id=pm.gnosis.heimdall)
```
Comment (tschubotz) [inline source/clients/android.md]: I would move these links inline to the sentence in line 5.

File: source/clients/android.md [hunk 4]
```diff
@@ -1,3 +1,26 @@
 # Android Client (Safe Personal Edition)
 
+## Install Mainnet Version
+
+The Mainnet version of the Gnosis Safe Android app can be downloaded from the Google PlayStore or from the GitHub releases page.
+
+Note: The versions from GitHub and the PlayStore are differently signed, therefore it is not possible to update the PlayStore version with a version downloaded from GitHub and vise-versa. Also the GitHub version will not recive automatic updates.
```
Comment (tschubotz) [inline source/clients/android.md]: viCe-versa

Comment (tschubotz) [reply source/clients/android.md]: recEive

File: source/clients/android.md [hunk 5]
```diff
@@ -1,3 +1,26 @@
...
+
+[View on PlayStore](https://play.google.com/store/apps/details?id=pm.gnosis.heimdall)
+[View on GitHub releases](https://github.com/gnosis/safe-android/releases) (apk file in the assets prefixed with `-release`)
```
Comment (tschubotz) [inline source/clients/android.md]: suffixed, not prefixed :)

File: source/clients/android.md (hunk 3 above)
Comment (tschubotz) [reply source/clients/android.md]: If you prefer them at the bottom, I would move them to 2 lines. They seem to be rendered in 1 which confused me 
image

File: source/clients/android.md [hunk 6]
```diff
@@ -1,3 +1,26 @@
...
+
+The Rinkeby version of the Gnosis Safe is only available from the GitHub releases page. This version will not receive automatic updates.
+
+[View on GitHub releases](https://github.com/gnosis/safe-android/releases) (apk file in the assets prefixed with `-rinkeby`)
+
+## Comments
+
+- To install the version from GitHub it might be required to allow installing applications from unknown sources.
+- The versions installed from GitHub, will not receive automatic updates.
+- For nightly versions please contact us on [Gitter](https://gitter.im/gnosis/Safe).
```
Comment (tschubotz) [inline source/clients/android.md]: better "nightly builds"?

File: source/clients/chrome_extension.md [hunk 7]
```diff
@@ -1,3 +1,34 @@
 # Chrome Extension (Safe Personal Edition)
 
+## Download Mainnet Version
+
+The Mainnet version of the Gnosis Safe browser extension can be downloaded from the Chrome Web Store or from the GitHub releases page.
+
+Note: The versions from GitHub and the Chrome Web Store are different, therefore it is not possible to update the Chrome Web Store version with a version downloaded from GitHub and vise-versa. Also the GitHub version will not recive automatic updates.
```
Comment (tschubotz) [inline source/clients/chrome_extension.md]: vice-versa and receive have typos

File: source/clients/chrome_extension.md [hunk 8]
```diff
@@ -1,3 +1,34 @@
...
+
+[View on Chrome Web Store](https://chrome.google.com/webstore/detail/gnosis-safe-smart-wallet/iecodoenhaghdlpodmhooppdhjhmibde)
```
Comment (tschubotz) [inline source/clients/chrome_extension.md]: I would put the 2 links in 2 lines here as well.

File: source/clients/chrome_extension.md [hunk 9]
```diff
@@ -1,3 +1,34 @@
...
+
+The Rinkeby version of the Gnosis Safe is only available from the GitHub releases page. This version will not receive automatic updates.
+
+[View on GitHub releases](https://github.com/gnosis/safe-browser-extension/releases) (zip file in the assets prefixed with `-rinkeby`)
+
+## Install Extension from GitHub
+
+1. Download the zip file for the corresponsing version from the [GitHub releases](https://github.com/gnosis/safe-browser-extension/releases)
+1. Unzip the file to your preferred location
+1. Open `chrome://extensions` in your Chrome web browser
```
Comment (tschubotz) [inline source/clients/chrome_extension.md]: Could this be a direct link?

File: source/services/notifications.md [hunk 10]
```diff
@@ -1,6 +1,9 @@
 # Notification Service
 Allows users to send signed transaction messages between devices taking part in the signing process.
 
+[Show on GitHub](https://github.com/gnosis/safe-notification-service)
+[Show on Swagger](https://safe-notification.gnosis.pm/)
```
Comment (tschubotz) [inline source/services/notifications.md]: This has the 2 line problems as well.

File: source/services/relay.md [hunk 11]
```diff
@@ -1,15 +1,10 @@
 # Transaction Relay Service
 This service allows us to have owners of the Safe contract that don’t need to hold any ETH on those owner addresses. How is this possible? The transaction relay service acts as a proxy, paying for the transaction fees and getting it back due to the transaction architecture we use.
 
-Our target user hold crypto in a centralized exchange (or on another Ethereum address) and wants to move it to a secure account. We don’t want the user to trust us, for moving the funds and deploying the smart contract on their behalf. We on the other side want to prevent users from spamming our services, there shouldn't be a need to trust the user either.
+Our target user hold crypto in a centralized exchange (or on another Ethereum address) and wants to move it to a secure account. We don’t want the user to trust us, for moving the funds and deploying the smart contract on their behalf. We on the other side want to prevent users from spamming our services, there shouldn't be a need to trust the user either. The process for this is descriped in the contracts deployment section.
```
Comment (tschubotz) [inline source/services/relay.md]: Should we link to the section directly?

File: source/clients/android.md (hunk 5 above)
Comment (rmeissner) [reply source/clients/android.md]: true ....

File: source/clients/chrome_extension.md (hunk 7 above)
Comment (rmeissner) [reply source/clients/chrome_extension.md]: copy pasta :D

**Pull request merged**